packages/core/src/infrastructure/database/migrations/2024-03-31T15:39:09Z-share-email-default.ts

```diff
@@ -0,0 +1,36 @@
+import { Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.updateTable('students').set({ share_email: true }).execute();
+
+  await db.schema
+    .alterTable('students')
+    .alterColumn('share_email', (column) => {
+      return column.setDefault(true);
+    })
+    .execute();
+
+  await db.schema
+    .alterTable('students')
+    .alterColumn('share_email', (column) => {
+      return column.setNotNull();
+    })
+    .execute();
+}
+
+export async function down(db: Kysely<any>) {
+  await db.schema
+    .alterTable('students')
+    .alterColumn('share_email', (column) => {
+      return column.dropNotNull();
+    })
+
+    .execute();
+
+  await db.schema
+    .alterTable('students')
+    .alterColumn('share_email', (column) => {
+      return column.dropDefault();
+    })
+    .execute();
+}
```

Comment:
Does it make more sense to combine share-email-default.ts with share-email.ts rather than have two separate migration files? Here I was following the example as seen with [birthday-notification](https://github.com/colorstackorg/oyster/blob/main/packages/core/src/infrastructure/database/migrations/2024-03-18T17%3A06%3A13Z-birthday-notification.ts) and [birthday-default.](https://github.com/colorstackorg/oyster/blob/main/packages/core/src/infrastructure/database/migrations/2024-03-18T19%3A45%3A26Z-birthday-default.ts). Not sure if the separate files are necessary

Reply:
Great question, yes please combine them! We only had to do that for the `birthday-notification`/`birthday-default` because we forgot to ship it with a default value, and so we needed to add it after.